pdc/apps/release/views.py

```diff
@@ -694,6 +694,10 @@ def create(self, request):
         if source_release.releasecomponent_set.count() == 0:
             return Response({'detail': 'there is no component in source release'},
                             status=status.HTTP_204_NO_CONTENT)
+        if not target_release.active:
+            return Response({'detail': 'can\'t clone components' +
+                                       'for target release %s with inactive' % target_release_id},
```

Comment:
Is it better to change the detail to "can't clone components to an inactive target release %s"